src/server/controllers/todos.js

```diff
@@ -24,6 +24,7 @@ exports.update = (request, response) => {
   Todo.findById(request.param('id')).then((todo) => {
     if (!todo) return response.status(404);
     todo.title = request.body.title;
+    todo.completed = !todo.completed;
```

Comment:
You are also negating `todo.completed` rather than `request.body.completed`, which means you will just continually toggle the object true/false rather than considering the actual input.